index.html

```diff
@@ -0,0 +1,555 @@
+                                <li><a href="#">Hackers are spreading fake Android Coronavirus trackers<br> to steal your bitcoins.</a></li>
+                                <li><a href="#">How to change your default search engine on Google Chrome</a></li>
+                                <li><a href="#">Google Play now has dark mode - here's how to turn it on.</a></li>
+                                <li><a href="#">Logitech built a 3-in-1 charger for Apple fans.</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="distract-info">
+                        <h1>Distract <i class="chevron right icon"></i></h1>
+                        <div class="distract-content">
+                            <h3>MOST POPULAR</h3>
+                            <p class="big-text">Instagram will place coronavirus advice<br> front and center on your feed.</p>
+                            <div class="meta">
+                              <small>Napier Lopez</small>
+                              <ul>
+                                <li>5 hours ago</li>
+                              </ul>  
+                            </div>
+                            
+                        </div>
+                        <div class="app-desc">
+                            <div>
+                                <h3>LATEST</h3>
+                                <p>Microsoft launches a coronavirus <br>tracker dashboard on Bing</p>
+                            </div>
+                            <img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/twitter-bot-pitching.png" alt="twitter-bot" />
+                        </div>
+                        <div class="app-list">
+                            <ul>
+                                <li><a href="#">Instagram will place coronavirus advice <br>front and center on your feed.</a></li>
+                                <li><a href="#">Hackers are spreading fake Android Coronavirus trackers<br> to steal your bitcoins.</a></li>
+                                <li><a href="#">How to change your default search engine on Google Chrome</a></li>
+                                <li><a href="#">Google Play now has dark mode - here's how to turn it on.</a></li>
+                                <li><a href="#">Logitech built a 3-in-1 charger for Apple fans.</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+
+                <section class="last-section">
+                    <header>
+                        <h2>Latest Deals <i class="chevron right icon"></i></h2>
+                        <p>See all Deals</p>
+                    </header>
+                    <div class="deals-flex">
+                        <div class="deals-item">
```

Comment:
- [ ] The elements in this section have a large image when in mobile screen size. 

- [ ] The title is not aligned correctly.

![Screenshot from 2020-03-18 12-16-16](https://user-images.githubusercontent.com/51926937/76993391-626f2380-6912-11ea-9afd-71139491c25a.png)

Original;
![Screenshot from 2020-03-18 12-16-10](https://user-images.githubusercontent.com/51926937/76993401-6733d780-6912-11ea-852a-aa609a7c71a1.png)